README.md

```diff
@@ -1,2 +1,145 @@
 # HelloMicroverse
-Project for module 1 day 2
+<!-- Project for module 1 day 2 -->
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> This project is about a excercise to understand Microverse tools
+
+**[Hello Microverse]** is a project to prctice whit the Microverse tools
+
+## 🛠 Built With <a name="built-with">Visual Code Editor</a>
+
+### Tech Stack <a name="tech-stack">HTML CSS</a>
+
+> HTML & CSS is markup metodology to undertand how is the web structured.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Code Editor</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Client PC</a></li>
+  </ul>
+</details>
+
```

Comment:
- [ ] Nice 👏  👍  you have added the Tech Stack of your project into README. But there are a few 🤌 misunderstandings 😅 I would like to clarify for you.

-   Visual studios is not a Client its a tool.
-    Client PC is not a Server in your case

> A client is a program that uses services that other programs provide. The programs that provide the services are called servers. The client makes a request for a service, and a server performs that service.

I suggest using `HTML/CSS` for your tech stack in this Project